### ISSUE TITLE
TODO: Remove EEPROM offset defines needing copied over

### ISSUE THREAD

**Issue by NachtRaveVL** [NachtRaveVL/Simple-Hydroponics-Arduino#6]
It really is pointless to do this when we can just store a 16-bit number indicating the size of the chunk at the start of the data block - essentially the creation of a more formalized chunk object. This would give us the ability to just simply define a dynamic chunk size, and have it also include things like, for instance, possibly a next-chunk #, where we can then string large segments together into a chunk-based system. Then we can do things like access the better EEPROM offsetting stuff by discounting the dynamic nature and moving to a formal static chunk size. We can still store chunks dynamically, but we can also better utilize the usage of the entire EEPROM by spreading the static chunks out using existing code in the EEPROM library we're already using.

Basically, let's do something better with our EEPROM data chunk logic to really take advantage of EEPROM writing/reading here. We can also just continue with this copy-over-the-output-of-this business. Not like we don't have SD cards/readers that are just as cheap and readily available.
